traffic_ops/app/lib/API/DeliveryServiceRegexes.pm

```diff
@@ -27,12 +28,18 @@ use Validate::Tiny ':all';
 sub all {
 	my $self = shift;
 
+	my $tenant_utils = Utils::Tenant->new($self);
+	my $tenants_data = $tenant_utils->create_tenants_data_from_db();
+
 	my $rs;
-	if ( &is_privileged($self) ) {
+	if ( &is_privileged($self) or $tenant_utils->ignore_ds_users_table()) {
```

Comment:
This code was written earlier...
BTW, I thought you were just concerned about having 2 separate parameters. Would you like to merge the "use-tenancy" and "ignore-ds-user-table" functions as well?